# default coordinate for the TaskDeliverSample action
x_delivery: 0.55
y_delivery: -0.3
z_delivery: 0.82  # was .78